include/SVInt.h

```diff
@@ -24,6 +24,8 @@ enum class LiteralBase : uint8_t {
     Hex
 };
 
+bool literalBaseFromChar(char base, LiteralBase& result);
```

Comment:
hmm, wasn't aware of the existence of std::optional. I think I will just leave it as the best solution is probably to use a smart enum with INVALID